### ISSUE TITLE
Don't login in version v2.0.0a5,  fbchat.NotLoggedIn: Password Forgotten account?

### ISSUE THREAD

**Issue by Ducvoccer** [carpedm20/fbchat#602]
I use v2.0.0a5 but i got issue like:
fbchat.NotLoggedIn: Password Forgotten account?

i sure, i correct email and password.
sorry about my english.

**Comment by xaadu** [comment]
Yeah, there is a login problem started from a3 to the current version and it's not fixed yet because of JavaScript validation in messenger login system! Look Here: #582
